docs/reference/how-to/size-your-shards.asciidoc

```diff
@@ -544,3 +544,50 @@ PUT _cluster/settings
   }
 }
 ----
+
+[discrete]
+==== Number of documents in the index shard cannot exceed [2147483519]
+
+NOTE: Historically, this error reported `Number of documents in the index shard cannot exceed [2147483519]` where 
+`index` was referencing a Lucene index which correlates to an <<glossary-shard,Elasticsearch shard>>. 
+
+Elasticsearch shards reflect Lucene's underlying https://github.com/apache/lucene/issues/5176[index 
+`MAX_DOC` hard limit] of 2,147,483,519 (`(2^31)-129`) docs. This figure calulates 
+from the sum of `docs.count` plus `docs.deleted` as reported by <<indices-stats,Index Stats>> 
+per shard. Attempting to surpass this limit, e.g. via ingestion's <<index-modules-translog,translog>>, 
+will cause error 
+
+[source, txt]
+----
+Elasticsearch exception [type=illegal_argument_exception, reason=Number of documents in the index shard cannot exceed [2147483519]]
+----
+
+or this error's sister stack trace will appear 
+
+[source, txt]
+----
+Caused by: java.lang.IllegalArgumentException: number of documents in the index cannot exceed 2147483519
```

Comment:
I don't see where this exact message could come from. Is it supposed to be the same as the previous one? I'm not sure we need both. I mean we could kind of say the same thing about every possible exception, but that doesn't sound like a good idea.

Reply:
It came up in a Support case. I was including for anyone doing a literal string search on Elastic's site / Google that this variation is missing the `[]`.

Reply:
Sure but (a) it should say `shard` not `index` now and (b) I don't see anywhere where we might omit the `[]` in practice so let's not omit them in the docs either. In any case, I'd much prefer to drop it here, I don't think it adds anything over the previous message and may cause more confusion than it solves.

Reply:
I agree I'd delete the second error for clarity's sake